src/main/java/moneytransferapp/MoneyTransferWorkflowImpl.java

```diff
@@ -0,0 +1,34 @@
+package moneytransferapp;
+
+import io.temporal.activity.ActivityOptions;
+import io.temporal.workflow.Workflow;
+ import io.temporal.common.RetryOptions;
+import java.time.Duration;
+
+// @@@SNIPSTART money-transfer-project-template-java-workflow-implementation
+public class MoneyTransferWorkflowImpl implements MoneyTransferWorkflow {
+
+    RetryOptions retryoptions =  RetryOptions.newBuilder()
```

Comment:
Seems like this might be worth commenting. As you know I'm still trying to decide if we should keep it.